tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/operator-finder/operator-finder.js

```diff
@@ -320,14 +394,15 @@ define(['jquery', 'lodash', 'log', 'handlebar', 'designViewUtils', 'app/source-e
         };
 
         /**
-         * Copies the syntax into the clipboard.
+         * copy the each clipboard syntax of extension.
          *
          * @param index Operator index
-         * @param container Current container to find the context.
+         * @param exIndex clipboard syntax
+         * @param container container Current container to find the context
          */
-        OperatorFinder.prototype.copyToClipboard = function (index, container) {
+        OperatorFinder.prototype.copyToClipboard = function (index, exIndex, container) {
             if (this._operators[index]) {
-                var syntax = buildSyntax(this._operators[index]);
+                var syntax = this._operators[index].syntax[exIndex].clipboardSyntax;
```

Comment:
Remove buildSyntax() function if is not used

Reply:
960249d this commit fix above request change